src/settings/ResourceTypesSettings.js

```diff
@@ -1,36 +1,34 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+import { FormattedMessage } from 'react-intl';
+
 import { ControlledVocab } from '@folio/stripes/smart-components';
 
 class ResourceTypesSettings extends React.Component {
   static propTypes = {
     stripes: PropTypes.shape({
       connect: PropTypes.func.isRequired,
-      intl: PropTypes.shape({
-        formatMessage: PropTypes.func.isRequired,
-      }).isRequired,
     }).isRequired,
   };
 
   constructor(props) {
     super(props);
+
     this.connectedControlledVocab = props.stripes.connect(ControlledVocab);
   }
 
   suppressEdit = term => term.source === 'rdacontent';
   suppressDelete = term => term.source === 'rdacontent';
 
   render() {
-    const { formatMessage } = this.props.stripes.intl;
-
     return (
       <this.connectedControlledVocab
```

Comment:
Please add columnMapping prop with the translations for the fields.
Please check all the occurrences for the connectedControlledVocab usage